### ISSUE TITLE
[Suggestion / Gameplay Issue] No clarification in-game whether 1M = 1mol/B or 1M = 1mol/0.001B

### ISSUE THREAD

**Issue by ThePixelist** [petrolpark/Destroy#317]
# Note
I'm a newbie to the mod

# Initial Situation
When looking at the molarity (M) required for mixtures for reactions, I couldn't tell what the units of M were in the context of buckets (B), ie how concentration related to B

# Problem
It takes experimentation to find out what the mod considers one litre, because 1 in-world bucket is 1m³, but it's also quite common for mods to analogise buckets with litres, and so it's unclear if the mod analogises buckets with L or m³.

# Suggestion(s):
- Specify somewhere* that 1M = 1mol/B or 1B = 1L
- \* Tooltips, ponder menu, JEI pages

**Comment by TheRedCrafter344** [comment]
It's currently inconsistent. 1B=1L when it comes to liquids, 1B=1m^3 when it comes to gases. This will be fixed in 1.0.

**Comment by petrolpark** [comment]
Yep, 1 bucket will be consistently 1m^3 in a future version. 1M will then be 1mol/L, as in real life.

**Comment by ThePixelist** [comment]
Will it be explained that 1B = 1000L?
